src/core/services.py

```diff
@@ -508,3 +508,55 @@ def queue_job(project_id: int, job_id: int, session: Session) -> None:
         logger.error(
             f"Cannot queue job {job_id}, it's of status {job.status()}."
         )
+
+
+def get_job_objects(
+    project_id: int, job_id: int, start: int, lenght: int
```

Comment:
```suggestion
    project_id: int, job_id: int, start: int, length: int
```

Reply:
This PR is based upon #72 and this is from there. It is all ready fixed in that PR.